### PR TITLE
Fix readme typo

### PR DESCRIPTION
**Pull request by mwanago**
The `sha256('🦄')` example provided in the README was actually a result of `sha1('🦄')`, I fixed that.
Also, I've added an example of how to use the library in the browser

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,11 +20,20 @@ npm install crypto-hash
...
 import {sha256} from 'crypto-hash';
 
 console.log(await sha256('🦄'));
-//=> '5df82936cbf0864be4b7ba801bee392457fde9e4'
+//=> '36bf255468003165652fe978eaaa8898e191664028475f83f506dabd95298efc'
+```
+
+### Browser
+```js
+import {sha256} from 'crypto-hash/browser';
```
Comment (sindresorhus) [inline readme.md:33]: This is invalid. There's no `browser` sub-export.

Comment (mwanago) [reply readme.md:33]: How should I use it in the browser then?
`import {sha256} from 'crypto-hash` throws an error for me, and `crypto-hash/browser` works fine.

```
Module parse failed: Unexpected token (4:50)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { Worker } from 'node:worker_threads';
| import crypto from 'node:crypto';
> const threadFilePath = new URL('thread.js', import.meta.url);
| let worker; // Lazy
```

Comment (sindresorhus) [reply readme.md:33]: This is really a problem with your bundler or bundler config. This package correctly implements ESM and package exports: https://nodejs.org/api/packages.html#conditional-exports

You are of course free to use `crypto-hash/browser` for now, but it's something that explicitly should not work, so your bundler might prevent it in the future.

Comment (sindresorhus) [reply readme.md]: This is not the place to ask Webpack questions :)

**Pull request merged**